scripts/cdnCheck.js

```diff
@@ -0,0 +1,16 @@
+import fetch from "node-fetch";
+import { dependencies } from "../package-lock.json";
+
+async function cdnCheck () {
+  const version = dependencies["@paypal/sdk-logos"].version;
+console.log(version);
+  fetch(`https://www.paypalobjects.com/js-sdk-logos/${version}/ideal-black.svg`)
```

Comment:
For readability, let's add `async` here to make it clear Node.js is going to wait for this call to finish.
```suggestion
  await fetch(`https://www.paypalobjects.com/js-sdk-logos/${version}/ideal-black.svg`)
```

Reply:
Also now that we have the paypal logos included, can we use the `paypal-default.svg` file instead of `ideal-black.svg`?